packages/flutter_tools/test/general.shard/ios/mac_test.dart

```diff
@@ -44,6 +44,58 @@ void main() {
       );
     });
 
+    group('startLogger', () {
+      late FakeProcessManager fakeProcessManager;
+
+      setUp(() {
+        fakeProcessManager = FakeProcessManager.empty();
+      });
+
+      testWithoutContext('starts idevicesyslog when USB connected', () async {
+        fakeProcessManager.addCommand(const FakeCommand(
+          command: <String>[
+            'HostArtifact.idevicesyslog', '-u', '1234'
+          ],
+          environment: <String, String>{'DYLD_LIBRARY_PATH': '/path/to/libraries'},
+        ));
```

Comment:
The Flutter Style Guide now recommends [avoiding the use of `setUp`](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#make-each-test-entirely-self-contained).

Try this instead:

```suggestion
      testWithoutContext('starts idevicesyslog when USB connected', () async {
        final FakeProcessManager fakeProcessManager = FakeProcessManager.list(
          <FakeCommand>[
            const FakeCommand(
              command: <String>['HostArtifact.idevicesyslog', '-u', '1234'],
              environment: <String, String>{
                'DYLD_LIBRARY_PATH': '/path/to/libraries'
              },
            ),
          ],
        );
```